README.md

```diff
@@ -1 +1,26 @@
-# Office-Space-Allocator
\ No newline at end of file
+# Office-Space-Allocator
+
+#### How should this be manually tested?
```

Comment:
A good way to start a README is by telling us what this project is and what it does. Then we can go on to how to set it up `locally`..
the word `manually` here may not sound professional maybe use `locally`